libs/application/templates/financial-aid/src/forms/Application.ts

```diff
@@ -39,7 +39,7 @@ export const Application: Form = buildForm({
         buildSubSection({
           condition: (_, externalData) =>
             ((externalData as unknown) as ExternalData).nationalRegistry?.data
-              ?.applicant?.spouse === undefined,
+              ?.applicant?.spouse === null,
```

Comment:
I did the same change for this here: https://github.com/island-is/island.is/pull/6810. But i used "==" instead of "===". The reason why I did that was I thought that the person who wrote the original code might have encountered that spouse was undefined sometime, so by using "== null" I am checking if spouse is null or undefined.